maintainers/maintainer-list.nix

```diff
@@ -6160,6 +6160,12 @@
     githubId = 20792;
     name = "Sebastian Galkin";
   };
+  parasrah = {
+    email = "nixos@parasrah.com";
+    github = "parasrah";
+    githubId = 14935550;
+    name = "Brad Pfannmuller";
+  };
```

Comment:
You should add yourself to the maintainers in a separate commit with the message

```
maintainers: add parasrah
```

Reply:
done :+1: